resources/assets/js/app/controllers/UserController.js

```diff
@@ -13,5 +14,23 @@ module.exports = Marionette.Object.extend({
         app.render(new UsersView({
             collection: users
         }));
+    },
+
+    showUserVotes: function(id) {
+
+        var showUsersVotesView = require('../views/votes/ShowUsersVotesView');
+        var usersVoteModel=require('../models/UsersVoteModel');
+        var usersVotesCollection=baseCollection.extend({
```

Comment:
you no need to create custom collection entities, for such case we have parentUrl, ask guys about this they will help you to optimize this